doc/source/DBEP/DBEP07.rst

```diff
@@ -0,0 +1,60 @@
+===================
+ DBEP 7 : Document mutation
+===================
+
+
+.. contents::
+   :local:
+
+Status
+======
+
+**Discussion**
+
+
+Branches and Pull requests
+==========================
+
+
+Abstract
+========
+The motivation of this DBEP is the handling of document mutation.
+There is a need for post-acquisition modification of metadata.
+
+
+
+
+
+Proposed Implementations
+====================
+There are three ways that have been discussed for handling this.
+Which one should be used has not been decided on as of now.
+
+
+#. Copy the original data into a backup data store. Create a new start document
```

Comment:
Changing the UID is :-1: .  We have promised that they will always be able to get the data back if they give us that string.